cves/kernel/CVE-2021-30002.yml

```
@@ -393,37 +396,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
     applies:
```

Just because you put false for everything else but kept this null
```suggestion
    applies: false
```